concent-builder/common_tasks/postrequire-configure-tasks.yml

```diff
@@ -0,0 +1,16 @@
+- name:    Install dependencies from testing
+  apt:  name={{ item }} default_release=testing
+  with_items:
```

Comment:
The refactoring of `concent-builder` playbooks should be in a separate commit. Please squash current fixups and separate the commits into smaller, independent changes.

Reply:
Done.